Assets/VRTK/Scripts/UI/VRTK_UICanvas.cs

```diff
@@ -108,16 +109,17 @@ protected virtual void SetupCanvas()
                 canvasRigidBody.isKinematic = true;
             }
 
-            CreateDraggablePanel(canvas, canvasSize);
+            StartCoroutine(CreateDraggablePanel(canvas, canvasSize));
```

Comment:
As this `StartCoroutine` is called from an `OnEnable` you'll need to make a reference to the coroutine and call `StopCoroutine` in the `OnDisable` method

e.g.

```
private Coroutine draggablePanelCreation;
...

draggablePanelCreation = StartCoroutine(CreateDraggablePanel(canvas, canvasSize));

....

protected virtual void OnDisable()
{
  StopCoroutine(createDraggablePanel);
}
```

Otherwise, if the panel is enabled and disabled before the next frame happens the coroutine will still attempt to run.